### PR TITLE
migrate proxy/userspace/proxier.go logs to structured logging

### PR DESCRIPTION
**Pull request by JornShen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

part of https://github.com/kubernetes/enhancements/issues/1602

**Which issue(s) this PR fixes**:

[keps/sig-instrumentation/1602-structured-logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging)

[Structured Logging migration instructions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/userspace/proxier.go [hunk 1]
```diff
@@ -389,17 +389,17 @@ func (proxier *Proxier) syncProxyRules() {
 	proxier.mu.Lock()
 	defer proxier.mu.Unlock()
 
-	klog.V(4).Infof("userspace proxy: processing %d service events", len(changes))
+	klog.V(4).InfoS("userspace proxy: processing service events", "changeNum", len(changes))
```
Comment (thockin) [inline pkg/proxy/userspace/proxier.go:392]: s/changeNum/count

**Review by thockin — COMMENTED** [review]
Just one small change and done.

/approve

File: pkg/proxy/userspace/proxier.go (hunk 1 above)
Comment (JornShen) [reply pkg/proxy/userspace/proxier.go]: done.

**Comment by JornShen** [comment]
updated. PTAL  :)
@thockin

**Comment by JornShen** [comment]
ping @smarterclayton @dcbw

**Comment by JornShen** [comment]
@thockin  we need a lgtm for this :)

**Comment by JornShen** [comment]
/cc @thockin  
Have a time to take a look ? 
It will be code freeze soon.

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**